provisioning/kubernetes/build.sh

```diff
@@ -1,6 +1,22 @@
 #!/usr/bin/env bash
 set -Eeuo pipefail
 
+# Etcd cluster for Templates API
+: "${ETCD_REPLICA_COUNT:=1}"
```

Comment:
detail - v docs vyssie pise ze `Number of replicas is defined by ETCD_REPLICA_COUNT env variable (default 3).`
podla tohto je ale default 1 nie?